sales/invoicing/down_payment.rst

```diff
@@ -2,44 +2,104 @@
+   Please note that if you choose **delivered quantities** as invoicing policy,
+   **you will not be able to create an invoice**.
+
+Basic sales flow using down payments
+====================================
 
-The first time you request a down payment you can select an income
-account and a tax setting that will be reused for next down payments.
+For this first example, we will use a 50% amount down payment with a product using an
+**ordered quantities** as invoicing policy. Be sure to check out our documentation about invoicing
+policies here: :doc:`invoicing_policy` to facilitate your use of this feature. When it comes to
+create and view the invoice, you will only have access to a draft invoice mentioning the down
+payment (as you can see below).
 
-.. image:: media/down_payment01.png
+.. image:: media/down_payment_2.png
    :align: center
+   :class: img-thumbnail
+   :alt: What is a basic sales flow using down payments on Odoo Sales (1)?
 
-You will then see the invoice for the down payment.
+There, you can post the invoice and register the payment of your customer. But, we all know that in
+real life this flow does not happen immediately. So, for now, you can return to the sales order.
+There, you will have the possibility to see the order as a customer with the **Customer preview**
+button or to reach easily the previous draft invoice with the **Invoice** button. In any case, the
+down payment will be mentioned on both (sales order and draft invoice).
 
-.. image:: media/down_payment02.png
+.. image:: media/down_payment_3.png
    :align: center
+   :class: img-thumbnail
+   :alt: What is a basic sales flow using down payments on Odoo Sales (2)?
 
-On the subsequent or final invoice, any prepayment made will be
-automatically deducted.
+To complete the flow, if the customer wants to pay the rest of his sales order, you must create
+another invoice. Once again, you will have the choice to make another down payment or to deduct all
+the down payments and so, paying the rest of the invoice as a regular invoice.
 
-.. image:: media/down_payment03.png
+.. image:: media/down_payment_4.png
    :align: center
+   :class: img-thumbnail
+   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?
+
+This flow is possible also with a down payment taking into account a fixed amount.
+
+.. important::
+   Be careful that if you do a down payment with a product using a **delivered quantities** as
+   invoicing policy, you won’t be able to deduct all the down payments when it comes to invoice your
+   customer. Indeed, you have to deliver a product before creating the final invoice. If nothing has
+   been delivered, you will create a **credit note** that will cancel the draft invoice created
+   after down payment. To do so, you have to install the **Inventory App** to confirm the delivery
+   or you have to enter manually the delivered quantity on the sales order.
```

Comment:
```suggestion
   customer. Indeed, you have to deliver a product before creating the final invoice. If nothing has
   been delivered, you create a **credit note** that cancels the draft invoice created
   after the down payment. To do so, you have to install the **Inventory App** to confirm the delivery.
   Otherwise, you can enter the delivered quantity manually on the sales order.
```